.github/renovate.json5

```
@@ -0,0 +1,50 @@
+{
+  $schema: "https://docs.renovatebot.com/renovate-schema.json",
+  extends: ["config:recommended", ":gitSignOff"],
+  prHourlyLimit: 10,
+  enabledManagers: ["github-actions", "pep621", "cargo", "pre-commit"],
+  "pre-commit": {
+    enabled: true
+  },
+  lockFileMaintenance: {
+    enabled: true,
+    automerge: true
+  },
+  configMigration: true,
+  labels: ["dependencies"],
+  schedule: ["every weekend"],
+  packageRules: [
+    {
+      matchManagers: ["github-actions"],
+      addLabels: ["github-actions"],
+      commitMessagePrefix: "⬆\uFE0F\uD83D\uDC68\u200D\uD83D\uDCBB"
+    },
+    {
+      matchManagers: ["pep621"],
+      addLabels: ["python"],
+      commitMessagePrefix: "⬆\uFE0F\uD83D\uDC0D"
+    },
+    {
+      matchManagers: ["cargo"],
+      addLabels: ["rust"],
+      commitMessagePrefix: "⬆\uD83E\uDD80"
```

For me, the arrow renders differently (not as an emoji `⬆️`, but in text mode `⬆`).
I think this should also include the emoji-style variant selector like the other messages.
```suggestion
      commitMessagePrefix: "⬆\uFE0F\uD83E\uDD80"
```

---

Ahh, thanks for catching that!